src/Expensify.js

```diff
@@ -182,7 +183,7 @@ function Expensify(props) {
     }
 
     return (
-        <>
+        <DeeplinkWrapper>
```

Comment:
Have you looked to see if we need to add this logic back in? https://github.com/infinitered/ExpensifyApp/pull/2/files#diff-ccbc9d5854dd4a9fa4e90ff053092cf02d44d109a75426d36b8ea72afd3613feL56

Reply:
The logic to handle if user is navigating from Old Dot was moved to the `Browser` component and the Wrapper is using that logic as well so I think we're good.